public/app/features/plugins/admin/components/PluginDetailsRightPanel.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```

Comment:
We don't longer need this line since https://github.com/grafana/grafana/pull/88802/